src/connection.rs

```diff
@@ -389,29 +389,37 @@ impl rustls_connection {
         }
     }
 
-    /// Retrieves the cipher suite agreed with the peer.
-    /// This returns NULL until the ciphersuite is agreed.
-    /// The returned pointer lives as long as the program.
+    /// Retrieves the cipher suite identifier agreed with the peer.
```

Comment:
Maybe say that the identifier is the standard IANA numbering and link to https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4 ?

Reply:
Good idea. Done.